samples/dnn/ssd_mobilenet_object_detection.cpp

```diff
@@ -86,16 +106,24 @@ int main(int argc, char** argv)
               cropSize);
 
     VideoWriter outputVideo;
-    outputVideo.open(parser.get<String>("out") ,
-                     static_cast<int>(cap.get(CV_CAP_PROP_FOURCC)),
-                     cap.get(CV_CAP_PROP_FPS), cropSize, true);
+    bool readSingleImage = !parser.get<String>("image").empty() && (int) cap.get(CV_CAP_PROP_FRAME_COUNT) == 1;
```

Comment:
I cannot do `bool readSingleImage = (int) cap.get(CV_CAP_PROP_FRAME_COUNT) == 1` as I got `VIDEOIO ERROR: V4L2: getting property #7 is not supported` when opening a camera stream.

Reply:
@catree, it seems to me a kind of complicated. Please try to use a single argument `video` as input for VideoCapture: `VideoCapture cap(parser.get<String>("video"));`. That works fine as for single image, set of images or for a video.

Reply:
Ok, I have removed the image option.